Clarify Kubernetes Support Lifecycle (#2013)
spectrocloud/librarium: docs/docs-content/integrations/microk8s.md

MicroK8s is a Cloud Native Computing Foundation (CNCF) certified upstream Kubernetes deployment that runs entirely on your workstation or edge device. It runs all Kubernetes services natively without virtual machines and packs all the required libraries and binaries.

### Support Lifecycle

> We support other Kubernetes distributions such as K3s, Microk8s, and RKE2 until their official EOL. The EOL is set by the respective owner. Once we stop supporting the minor version, we initiate the deprecation process. You can learn more about our Kubernetes support lifecycle in the [Kubernetes Support Lifecycle](./kubernetes-support.md#other-kubernetes-distributions) page.

Reviewer: ```suggestion
We support other Kubernetes distributions such as K3s, Microk8s, and RKE2 until their official EOL. The EOL is set by the respective owner. Once we stop supporting the minor version, we initiate the deprecation process. Learn more on the [Kubernetes Support Lifecycle](./kubernetes-support.md#other-kubernetes-distributions) page.
```
